src/vs/workbench/contrib/terminal/browser/terminalInstanceService.ts

```diff
@@ -24,10 +24,10 @@ let WebglAddon: typeof XTermWebglAddon;
 export class TerminalInstanceService extends Disposable implements ITerminalInstanceService {
 	public _serviceBrand: undefined;
 
-	private readonly _onPtyHostExit = this._register(new Emitter<void>());
-	readonly onPtyHostExit = this._onPtyHostExit.event;
-	private readonly _onPtyHostUnresponsive = this._register(new Emitter<void>());
-	readonly onPtyHostUnresponsive = this._onPtyHostUnresponsive.event;
+	readonly onPtyHostExit = Event.None;
```

Comment:
are these needed? and these?
https://github.com/microsoft/vscode/blob/f9e6b352d1cf054af74deb1bacb03e255ea4c52a/src/vs/workbench/contrib/terminal/browser/terminal.ts#L32-L48

or to do items?

Reply:
They're required in the interface so we need to fill them in here. I only discovered `Event.None` recently which is much nicer than including real events.